### PR TITLE
EAS-2041 Add LoopbackResponse reset and tear-down step to clear test blackout

### PR DESCRIPTION
**Pull request by jonathan-owens-gds**
Uses an alarm blackout, similar to the scheduled lambda blackouts, to prevent alarms being raised during the functional test runs.

### CODE REVIEW

File: .codepipeline/buildspec-tests-build.yml [hunk 1]
```diff
@@ -52,13 +46,13 @@ phases:
   build:
     on-failure: CONTINUE
     commands:
-      - make test-broadcast-flow; exit 0
+      # - make test-broadcast-flow; exit 0
```
Comment (gds-gharper) [inline .codepipeline/buildspec-tests-build.yml:49]: Why comment out the code, if it's being left for a reason add a comment to say why it's commented out?

File: tests/functional/preview_and_dev/test_cbc_integration.py [hunk 2]
```diff
@@ -67,11 +42,10 @@ def test_get_loopback_request_with_bad_id_returns_no_items():
     assert len(response["Items"]) == 0
 
 
-@pytest.mark.skip("Skip while vodafone loopback architecture issue is investigated")
+# @pytest.mark.skip("Skip while vodafone loopback architecture issue is investigated")
```
Comment (gds-gharper) [inline tests/functional/preview_and_dev/test_cbc_integration.py:45]: If we no longer need to skip we should remove this

File: tests/functional/preview_and_dev/test_cbc_integration.py [hunk 3]
```diff
@@ -199,22 +186,24 @@ def test_broadcast_with_az1_failure_tries_az2(driver, api_client):
...
         responses, "MnoName", secondary_cbc, "ResponseCode"
     )
-    assert o2_az2_response_code == success_code
+    assert az2_response_code == "200"
 
     cancel_alert(driver, broadcast_id)
 
 
-@pytest.mark.skip("Skip while vodafone loopback architecture issue is investigated")
+# @pytest.mark.skip("Skip while vodafone loopback architecture issue is investigated")
```
Comment (gds-gharper) [inline tests/functional/preview_and_dev/test_cbc_integration.py:206]: Remove commented out code

**Review by gds-gharper — DISMISSED** [review]

File: tests/functional/preview_and_dev/test_cbc_integration.py (hunk 3 above)
Comment (jonathan-owens-gds) [reply tests/functional/preview_and_dev/test_cbc_integration.py]: comments removed

File: tests/functional/preview_and_dev/test_cbc_integration.py (hunk 2 above)
Comment (jonathan-owens-gds) [reply tests/functional/preview_and_dev/test_cbc_integration.py]: comment removed

**Review by Adam-C-Mitchell — APPROVED** [review]
This looks good to me,

**Pull request merged**